doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1140,21 +1218,27 @@ A context registration is represented by a JSON object with the following fields
 * `subject` : It s an object that describes the subject of the registration.
 * `callback` : URL of the service which is registered. In the case of a Context Provider
 corresponds to the URL of the provider service.
-* `options` : It is a dictionary which can contain different parameters related to the registration, namely:
-  * `duration`: Duration of the registration in ISO8601 format. Default duration is infinite.
+* `duration`: Duration of the registration in ISO8601 format. Default duration is infinite.
 
 A `subject` contains the following subfields:
 
+* `id` : Id of the affected entities (optional). It cannot be use at the same time than `idPattern`.
+* `idPattern` : Id pattern of the affected entities (optional). It cannot be use at the same time than `id`.
 * `entity` : Id or pattern id which identifies the affected entities (optional).
 * `type` : Type of the affected entities (optional).
 * `attributes`: List of attributes to be provided (if not specified it would mean all).
 
-Entity or type must be present.
+`id`/`idPattern` or `type` must be present.
```

Comment:
Given that id/idPattern and type are described in different bullets, this the very first place in the text where I can describe restrictions applying both...